target/linux/x86/image/Makefile

```diff
@@ -134,6 +134,12 @@ ifneq ($(CONFIG_VMDK_IMAGES),)
 	$(STAGING_DIR_HOST)/bin/qemu-img convert -f raw -O vmdk \
 		$(BIN_DIR)/$(IMG_COMBINED)-$(1).img \
 		$(BIN_DIR)/$(IMG_COMBINED)-$(1).vmdk
+	#build the ESXI VMDK with 
+	rm $(BIN_DIR)/$(IMG_COMBINED)-$(1)-esxi.vmdk || true
+	rm $(BIN_DIR)/$(IMG_COMBINED)-$(1)-esxi-flat.vmdk || true
```

Comment:
Why do you delete this file?

Reply:
This is following the pattern of the other image Build definitions to remove the output files before attempting the qemu-img convert process. The monolithic flat vmdk disk type requires a descriptor file (*-esxi.vmdk) as well as the flat image file (*-esxi-flat.vmdk). Both of these files must be loaded into the ESXI data store for a valid VMDK image to be available.

Reply:
where is the *-esxi-flat.vmdk file created?

Reply:
The output of qemu-img convert to monolithicFlat produces both the descriptor file (-esxi.vmdk) and the image file (-esxi-flat.vmdk).